packages/bpk-mixins/src/mixins/_chips.scss

```diff
@@ -24,158 +24,215 @@ $bpk-spacing-v2: true;
   padding: 0 bpk-spacing-base();
   align-items: center;
   border: none;
+  border-radius: $bpk-border-radius-xs * 2;
 
-  @include bpk-border-radius-pill;
+  svg {
+    /* stylelint-disable declaration-no-important */
+    width: 16 * $bpk-one-pixel-rem !important;
+    height: 16 * $bpk-one-pixel-rem !important;
+  }
+
+  &__leading-accessory-view {
+    display: inline-flex;
+    fill: currentColor;
+
+    @include bpk-margin-trailing(bpk-spacing-md());
+  }
+
+  &__trailing-accessory-view {
+    display: inline-flex;
+    fill: currentColor;
+
+    @include bpk-margin-leading(bpk-spacing-md(), false);
+    @include bpk-margin-trailing(-1 * bpk-spacing-md(), false);
+  }
 
-  &--light {
-    background-color: $bpk-color-white;
-    color: $bpk-color-sky-gray;
+  &--on-dark {
+    background-color: transparent;
+    color: $bpk-text-on-dark-day;
 
-    @include bpk-border-sm($bpk-color-sky-gray-tint-03);
+    @include bpk-border-sm($bpk-line-on-dark-day);
 
     @include bpk-hover {
-      background-color: $bpk-color-sky-gray-tint-07;
+      @include bpk-border-sm($bpk-surface-default-day);
     }
 
     &:active:not(:disabled) {
-      background-color: $bpk-color-sky-gray-tint-06;
+      @include bpk-border-sm($bpk-surface-default-day);
     }
 
     &-selected {
       @include bpk-themeable-property(
         color,
-        --bpk-chip-light-selected-text-color,
-        $bpk-color-white
+        --bpk-chip-on-dark-selected-text-color,
+        $bpk-text-on-light-day
       );
       @include bpk-themeable-property(
         background-color,
-        --bpk-chip-light-selected-background-color,
-        $bpk-color-sky-blue
+        --bpk-chip-on-dark-selected-background-color,
+        $bpk-surface-default-day
       );
       @include bpk-border-sm(transparent);
 
       @include bpk-hover {
-        @include bpk-themeable-property(
-          background-color,
-          --bpk-chip-light-selected-hover-background-color,
-          $bpk-color-sky-blue-shade-01
-        );
+        @include bpk-border-sm(transparent);
       }
 
       &:active:not(:disabled) {
-        @include bpk-themeable-property(
-          background-color,
-          --bpk-chip-light-selected-active-background-color,
-          $bpk-color-sky-blue-shade-02
-        );
+        @include bpk-border-sm(transparent);
+      }
+    }
+
+    &-dismissible {
+      @include bpk-hover {
+        svg {
+          fill: currentColor;
```

Comment:
I think the way `currentColor` works is that it takes the colour which is set on `color` property even after the `color` changed so this wouldn't be needed as it's already set in the `leading-accessory-view` and `trailing-accessory-view` mixins